data/txts/tips/general_game.lua

```diff
@@ -62,7 +62,7 @@ tips = {
       seconds = 4
    },
    {
-      text = _("An economy consists of all buildings and flags connected by roads. The supplies and demands of buildings are handled in its economy."),
+      text = _("An economy consists of all buildings and flags connected by roads, ferries or ship lines. The supplies and demands of buildings are handled in its economy."),
```

Comment:
I think "ferries" is OK. How about this:

```suggestion
      text = _("An economy consists of all buildings and flags connected by roads, ferries or ships. The supplies and demands of buildings are handled in its economy."),
```